neo/Helper.cs

```diff
@@ -320,5 +320,34 @@ public static IConfigurationRoot LoadConfig(string config)
                 .AddJsonFile(configFile, true)
                 .Build();
         }
+
+        public static T StrictEnum<T>(int value) where T : Enum
+        {
```

Comment:
If is not used, we should remove this method

Reply:
I thought it would be good to have it already there for the moment somebody needs it.

Reply:
I think that all of our enums will be `byte`